### PR TITLE
syscontainers: Add more docstrings

### PR DESCRIPTION
**Pull request by ashcrow**
## Description
Added more docstrings for ``syscontainers.py``

### CODE REVIEW

File: Atomic/syscontainers.py [hunk 1]
```diff
@@ -1088,7 +1189,7 @@ def _do_checkout(self, repo, options):
         if not remote_path:
             rootfs_fd = None
             try:
-                rootfs_fd = os.open(rootfs, os.O_DIRECTORY)
+                rootfps_fd = os.open(rootfs, os.O_DIRECTORY)
```
Comment (giuseppe) [inline Atomic/syscontainers.py]: Looks like a typo here

File: Atomic/syscontainers.py [hunk 2]
```diff
@@ -2542,6 +2792,19 @@ def _rewrite_config_args(self, orig_config, dest_config, args, tty=None, checkou
             return config
 
     def container_exec(self, name, detach, args):
+        """
+        Run exec on a container.
+
+        :param name: The name of the container.
+        :type name: str
+        :param detach: If detatch should be used.
```
Comment (giuseppe) [inline Atomic/syscontainers.py]: Detatch

**Comment by ashcrow** [comment]
Updated, thanks @giuseppe

**Comment by ashcrow** [comment]
@rh-atomic-bot retry

**Comment by ashcrow** [comment]
bot, retest this please

**Comment by ashcrow** [comment]
FWIW this is a doc only update.

**Comment by giuseppe** [comment]
@rh-atomic-bot r+ 49b3c13

**Comment by ashcrow** [comment]
@rh-atomic-bot retry

**Pull request closed without merging**